### PR TITLE
*: cut V0.6.2 release

### PR DESCRIPTION
**Pull request by indradhanush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+### Bug fixes
+
+- Fix persisted example [#254](https://github.com/habitat-sh/habitat-operator/pull/254)
+- Makefile: Fix clean-test target [#287](https://github.com/habitat-sh/habitat-operator/pull/287)
+- examples: Fix incorrect mount path for persistent [#286](https://github.com/habitat-sh/habitat-operator/pull/286)
+
+### Features & Enhancements
+
+- Add .bldr.toml to get automatic rebuilds on merged PR's [#256](https://github.com/habitat-sh/habitat-operator/pull/256)
+- add channel to supervisor [#259](https://github.com/habitat-sh/habitat-operator/pull/259)
```
Comment (krnowak) [inline CHANGELOG.md]: I'd keep this. Capitalize "a" in "add". Maybe reword it to make it clearer what it is about. "Add Habitat package channel label to CRD"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+- Kubernetes 1.10 [#258](https://github.com/habitat-sh/habitat-operator/pull/258)
```
Comment (krnowak) [inline CHANGELOG.md]: I'd keep this. Maybe reword it to "Add support for Kubernetes 1.10, and drop it for 1.7"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+- Broadcast events [#267](https://github.com/habitat-sh/habitat-operator/pull/267)
```
Comment (krnowak) [inline CHANGELOG.md]: I'd keep this. Not sure about how to reword it, though.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,50 @@
...
+- Remove dependency on pflag library [#271](https://github.com/habitat-sh/habitat-operator/pull/271)
```
Comment (krnowak) [inline CHANGELOG.md]: I'd keep this.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,50 @@
...
+- Compare Resource Versions on update [#272](https://github.com/habitat-sh/habitat-operator/pull/272)
```
Comment (krnowak) [inline CHANGELOG.md]: This too.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,50 @@
...
+- Mark item done after validity check [#274](https://github.com/habitat-sh/habitat-operator/pull/274)
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this one too, but I think it should be a bug fix.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,50 @@
...
+- controller: print custom version string instead of pointer [#275](https://github.com/habitat-sh/habitat-operator/pull/275)
```
Comment (krnowak) [inline CHANGELOG.md]: Bug fix. And while rewording, please drop the "controller: " prefix and capitalize "p" in "print".

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,50 @@
...
+- Dockerfile: start from scratch [#277](https://github.com/habitat-sh/habitat-operator/pull/277)
+- Makefile: Remove dependency on minikube [#288](https://github.com/habitat-sh/habitat-operator/pull/288)
+- Remove Travis [#291](https://github.com/habitat-sh/habitat-operator/pull/291)
+- CI: Skip tests [#289](https://github.com/habitat-sh/habitat-operator/pull/289)
+- Use labels on examples [#284](https://github.com/habitat-sh/habitat-operator/pull/284)
+- Support double signal handling [#285](https://github.com/habitat-sh/habitat-operator/pull/285)
+- CI: Fix CI build [#296](https://github.com/habitat-sh/habitat-operator/pull/296)
+- CI: Manually fix CI build [#297](https://github.com/habitat-sh/habitat-operator/pull/297)
+- example: Add patch to events in rbac example [#300](https://github.com/habitat-sh/habitat-operator/pull/300)
+- Run E2E tests on GCE [#302](https://github.com/habitat-sh/habitat-operator/pull/302)
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,50 @@
...
+- Enable test for persistent storage [#303](https://github.com/habitat-sh/habitat-operator/pull/303)
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,50 @@
...
+- Run e2e tests on multiple versions of kubernetes [#306](https://github.com/habitat-sh/habitat-operator/pull/306)
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this.

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,50 @@
...
+- Delete pods if StatefulSet object is updated [#307](https://github.com/habitat-sh/habitat-operator/pull/307)
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this.

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,50 @@
...
+### Docs
+
+- docs: Add instructions to update & verify scripts [#255](https://github.com/habitat-sh/habitat-operator/pull/255)
+- doc: Add design document [#253](https://github.com/habitat-sh/habitat-operator/pull/253)
+
+
+### Deprecations
+
+- CHANGELOG: Deprecate customVersion = v1beta1 [#257](https://github.com/habitat-sh/habitat-operator/pull/257)
+- This is the last version to support v1beta1 and it will be dropped in the next release.
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this. Reword it maybe to "This is the last version to support v1beta1 custom version of Habitat CRD."

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,50 @@
...
+- Compare Resource Versions on update [#272](https://github.com/habitat-sh/habitat-operator/pull/272)
+- Mark item done after validity check [#274](https://github.com/habitat-sh/habitat-operator/pull/274)
+- controller: print custom version string instead of pointer [#275](https://github.com/habitat-sh/habitat-operator/pull/275)
+- Rename and comment operator manifest [#278](https://github.com/habitat-sh/habitat-operator/pull/278)
+- Dockerfile: start from scratch [#277](https://github.com/habitat-sh/habitat-operator/pull/277)
+- Makefile: Remove dependency on minikube [#288](https://github.com/habitat-sh/habitat-operator/pull/288)
+- Remove Travis [#291](https://github.com/habitat-sh/habitat-operator/pull/291)
+- CI: Skip tests [#289](https://github.com/habitat-sh/habitat-operator/pull/289)
+- Use labels on examples [#284](https://github.com/habitat-sh/habitat-operator/pull/284)
+- Support double signal handling [#285](https://github.com/habitat-sh/habitat-operator/pull/285)
```
Comment (krnowak) [inline CHANGELOG.md]: Keep this. And maybe reword it as "Shutdown gracefully through double signal handling".

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,50 @@
...
+- example: Add patch to events in rbac example [#300](https://github.com/habitat-sh/habitat-operator/pull/300)
+- Run E2E tests on GCE [#302](https://github.com/habitat-sh/habitat-operator/pull/302)
+- Enable test for persistent storage [#303](https://github.com/habitat-sh/habitat-operator/pull/303)
+- Run e2e tests on multiple versions of kubernetes [#306](https://github.com/habitat-sh/habitat-operator/pull/306)
+- Delete pods if StatefulSet object is updated [#307](https://github.com/habitat-sh/habitat-operator/pull/307)
+
+### Docs
+
+- docs: Add instructions to update & verify scripts [#255](https://github.com/habitat-sh/habitat-operator/pull/255)
+- doc: Add design document [#253](https://github.com/habitat-sh/habitat-operator/pull/253)
```
Comment (krnowak) [inline CHANGELOG.md]: Maybe keep this one, not sure…

**Comment by krnowak** [comment]
Also, not sure whether to make it `0.6.2` or rather `0.7.0`. We add support for k8s 1.10 here. Which is a shame, we should have done a release that does this way earlier…

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,39 @@
...
+
+### Features & Enhancements
+
+- Add "Channel" to Habitat CRD for Habitat packages [#259](https://github.com/habitat-sh/habitat-operator/pull/259)
+- Add support for Kubernetes 1.10 [#258](https://github.com/habitat-sh/habitat-operator/pull/258)
+- Broadcast events when Habitat objects are modified or fail validation [#267](https://github.com/habitat-sh/habitat-operator/pull/267)
+- Remove dependency on pflag library [#271](https://github.com/habitat-sh/habitat-operator/pull/271)
+- Compare Resource Versions on update [#272](https://github.com/habitat-sh/habitat-operator/pull/272)
+- Shutdown gracefully through double signal handling [#285](https://github.com/habitat-sh/habitat-operator/pull/285)
+- Run E2E tests on GCE [#302](https://github.com/habitat-sh/habitat-operator/pull/302)
```
Comment (krnowak) [inline CHANGELOG.md]: Either turn this "E2E" into "e2e" or in 2 lines below, "e2e" to "E2E" for consistency.

File: doc/release-process.md [hunk 16]
```diff
@@ -9,7 +9,11 @@ Then run the following command to update version across files:
 
     make update-version
 
-Document the changes in this release in the `CHANGELOG.md` file, following the already established pattern. Commit all the changes and `CHANGELOG.md` file under one commit message e.g.: `*: cut 0.6.1 release`. Create a PR with the changes.
+Document the important changes in this release in the `CHANGELOG.md` file, following the already established pattern. You can get a list of changes since the last release by running:
+
+    git log --merges v0.6.2...master --reverse
+
+Commit all the generated changes and those to the `CHANGELOG.md` file under one commit message e.g.: `*: cut 0.6.2 release`. Create a PR with the changes.
```
Comment (krnowak) [inline doc/release-process.md]: Commit all the generated changes and the changes in the `CHANGELOG.md` file under one…

I think that reads a bit better.

**Comment by indradhanush** [comment]
Closing this in favor of https://github.com/habitat-sh/habitat-operator/pull/313 which is an identical branch but with the right name according to the release version, i.e. `v0.7.0`.
